test/yaml/vm_with_persistedVolumes/vmkey

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
what is this private key for? is it OK to expose to anyone?

Reply:
this is the keypair ONLY used in the test to access to the VMs and only valid if the pubkey is specified in the test yaml. it is OK to make it public -- in fact, they are taken from virtlet project anyways.